src/control/security/grpc_authorization.go

```diff
@@ -39,19 +39,25 @@ func (c Component) String() string {
 
 // methodAuthorizations is the map for checking which components are authorized to make the specific method call.
 var methodAuthorizations = map[string][]Component{
-	"/ctl.MgmtCtl/StoragePrepare":     {ComponentAdmin},
-	"/ctl.MgmtCtl/StorageScan":        {ComponentAdmin},
-	"/ctl.MgmtCtl/StorageFormat":      {ComponentAdmin},
-	"/ctl.MgmtCtl/SystemQuery":        {ComponentAdmin},
-	"/ctl.MgmtCtl/SystemStop":         {ComponentAdmin},
-	"/ctl.MgmtCtl/SystemResetFormat":  {ComponentAdmin},
-	"/ctl.MgmtCtl/SystemStart":        {ComponentAdmin},
-	"/ctl.MgmtCtl/NetworkScan":        {ComponentAdmin},
-	"/ctl.MgmtCtl/FirmwareQuery":      {ComponentAdmin},
-	"/ctl.MgmtCtl/FirmwareUpdate":     {ComponentAdmin},
+	"/ctl.CtlSvc/StoragePrepare":      {ComponentAdmin},
```

Comment:
I don't think it's a problem per-se, but it would be a nice improvement if this list were sorted alphabetically unless there's another reason for the specific order?

Reply:
I think they were added in logical groupings but to be honest I wasn't paying close attention to that when I reordered things. We can reorder them later if it makes sense.